sdv/constraints/utils.py

```diff
@@ -104,3 +104,60 @@ def sigmoid(data, low, high):
     data = data * (high - low) + low
 
     return data
+
+
+def check_nans_row(row):
```

Comment:
I put the following functions in `utils.py` if we want to re-use them for other constraints (ex: `RangeConstraint`)